cmd/otelcol/config/collector/full_config_linux.yaml

```
@@ -580,6 +622,6 @@ service:
       processors: [memory_limiter, batch]
       exporters: [signalfx]
     logs:
-      receivers: [fluentforward, otlp]
+      receivers: [fluentforward, otlp, filelog]
```

I would recommend that we revert this line change for now and land the rest of the contribution.
```suggestion
      receivers: [fluentforward, otlp]
```